modules/developer-cli-odo-creating-and-deploying-a-nodejs-application-with-odo.adoc

```diff
@@ -60,7 +60,10 @@ $ cd <directory_name>
 $ odo create nodejs
 ----
 +
-NOTE: By default, the latest image is used. You can also explicitly specify an image version by using `odo create openshift/nodejs:8`.
+[NOTE]
+====
+By default, the latest image is used. You can also explicitly specify an image version by using `odo create openshift/nodejs:8`.
```

Comment:
Do you think the `odo create openshift/nodejs:8` command should be in its own code block similar to the formating in the NOTE [here](https://github.com/openshift/openshift-docs/pull/52108/files#diff-1451c10ec7a199273aa84e11f0923335cd6a92cf78b472b5c6fb446115cf3524R246). I wasn't sure if the formating was different based on the command.

Reply:
It probably could be, but I think it's acceptable to also put the command inline like this, since it's in a note. So I'll probably leave it stay as is, since I'm just worrying about the note syntax. Thanks!